docs/fastdds/use_cases/large_data/large_data.rst

```diff
@@ -258,8 +258,12 @@ Example configuration
 
    .. tab:: XML
 
-      There is currently no way of configuring flow controllers with XML.
-      This will be added in future releases of the product.
+      .. literalinclude:: /../code/XMLTester.xml
+         :language: xml
+         :start-after: <!-->CONF-QOS-FLOWCONTROLLER
+         :end-before: <!--><-->
+         :lines: 2-3, 5-25
```

Comment:
We missed the data writer configuration in the original PR too ([see this PR generated docs]() and [the original PR docs]()).
I am fixing the master snippet here:
- #794

```suggestion
         :lines: 2-3, 5-
```